src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/UpdateSpecificMethodologyAuthorizationHandlersTests.cs

```diff
@@ -1,20 +1,42 @@
 ﻿using System.Threading.Tasks;
 using GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
 using Xunit;
 using static GovUk.Education.ExploreEducationStatistics.Admin.Security.SecurityClaimTypes;
 using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Security.AuthorizationHandlers.Utils.AuthorizationHandlersTestUtil;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.MethodologyStatus;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Security.AuthorizationHandlers
 {
     public class UpdateSpecificMethodologyAuthorizationHandlersTests
     {
         [Fact]
-        public async Task UpdateAllSpecificMethodologies()
+        public async Task UpdateAllSpecificMethodologies_DraftMethodology()
         {
-            await AssertHandlerSucceedsWithCorrectClaims<UpdateSpecificMethodologyRequirement>(
+            var draftMethodology = new Methodology
+            {
+                Status = Draft
+            };
+            
+            await AssertHandlerSucceedsWithCorrectClaims<Methodology, UpdateSpecificMethodologyRequirement>(
                 new UpdateSpecificMethodologyAuthorizationHandler(),
+                draftMethodology,
                 UpdateAllMethodologies
             );
         }
+
+        [Fact]
+        public async Task UpdateAllSpecificMethodologies_ApprovedMethodologyFails()
+        {
+            var approvedMethodology = new Methodology
```

Comment:
See comment on the Authorization handler about this check being used to update a methodology which is also used by the call to unapprove a methodology. Due to its RESTful design there's only one update method which serves multiple purposes including unapproving. At least while we don't have amendments unapproving will still be necessary.

Reply:
Cool yeah will retest around this, and no doubt there will need to be a fix put in place.  Now it's a bit more obvious why this status check wasn't in the handler in the first place.

Not sure what your thoughts are on this RESTful approach?  I could imagine you could spend quite a bit of time validating that the updates being made were valid combinations - for instance, should you be able to change a bunch of content and approve in the same call?  Could be argued both ways.  Personally although it's more boilerplate, I do prefer the explicit individual calls, especially if doing something important like updating the status, and I def prefer the idea of an individual handler for approval workflow stuff.

Reply:
I prefer the explicit calls too, as the update methods for methodology and releases are having to do quite a lot. Despite the way we have designed the UI's into separate tabs, currently we still need to worry about the update/validation/security of fields on other tabs that are never getting modified which seems pointless and can lead to bugs like this.

Reply:
Rolled back behaviour and added a TODO